Gemfile.lock

```diff
@@ -202,7 +202,7 @@ GEM
     mini_portile2 (2.8.1)
     minitest (5.18.0)
     msgpack (1.7.0)
-    multipart-post (2.3.0)
+    multipart-post (2.2.3)
```

Comment:
This minor version change completely alters the require structure of the gem so breaks its faraday integration